models/issue_comment.go

```diff
@@ -264,6 +277,23 @@ func (c *Comment) LoadAssignees() error {
 	return nil
 }
 
+// LoadCommitStatus if comment.Type is CommentTypePullPushCommit, then load commit status
+func (c *Comment) LoadCommitStatus() error {
+	if c.CommitStatus == nil {
```

Comment:
reverse it?

e.g.
```go
if c.CommitStatus != nil {
    return nil
}
```

Then everything else can be placed on just one line of indentation.